ci/buildspec_deploy_main.yml

```diff
@@ -0,0 +1,19 @@
+# AWS CodeBuild config
+# This file is referenced in the infra repo as a CodeBuild buildspec target.
+
+# This 1 variable is set in the codebuild project definition's environment secion.
+# SDLC_STAGE
+
+version: 0.2
+env:
+  shell: bash
+phases:
+  install:
+    runtime-versions:
+      nodejs: 16
+
+  build:
+    commands:
+      - aws ecs update-service --force-new-deployment --cluster "doorway-${SDLC_STAGE}-srv-backend" --service "doorway-${SDLC_STAGE}-srv-backend"
```

Comment:
It's generally preferable to take the full value from config rather than hard code some or all of the value in.  One example why we would want to do this is that a future fix for the infra TF (time permitting) is to group all of the services into a single cluster.  Making a quick change to a tfvars file is generally safer and faster than making changes to a buildspec or *.tf file.

Another thing to remember is that this repo is open source and is (theoretically) going to be used by more than just MTC/Exygy for the Doorway project, so it's entirely possible that someone may deploy this with a project ID other than "doorway" (the possibility of a "bloom" comes to mind).  Exygy intends to bring changes from both the `doorway-infra` and `doorway` repos back into `bloom-infra` and `bloom` repos, so we want to make sure to keep things as portable and flexible as possible.

This isn't blocking since we should be updating this in the near future to replace with CodeDeploy, but just something to keep in mind.